utils/bandwidthd/Makefile

```diff
@@ -48,14 +48,14 @@ define Package/bandwidthd-sqlite
 $(call Package/bandwidthd/Default)
   TITLE += (with sqlite enabled)
   VARIANT:=sqlite
-  DEPENDS:=+libgd +libpcap +libsqlite3 +php7 +php7-cgi +php7-mod-gd +php7-mod-pdo +php7-mod-pdo-sqlite
+  DEPENDS:=+libgd +libpcap +libsqlite3 +php8 +php8-cgi +php8-mod-gd +php8-mod-pdo +php8-mod-pdo-sqlite
 endef
 
 define Package/bandwidthd-php
 $(call Package/bandwidthd/Default)
   TITLE = PHP files to graph bandwidthd data from postgresql
   VARIANT:=php
-  DEPENDS:=+libpcre +libxml2 +php7 +php7-cgi +php7-mod-pgsql +php7-mod-gd
+  DEPENDS:=+libpcre +libxml2 +php8 +php8-cgi +php8-mod-pgsql +php8-mod-gd
 endef
 
 define Package/bandwidthd/description
```

Comment:
it is better do not use `@ someone` in commit message

Reply:
Why?